subsys/bluetooth/audio/bap_broadcast_sink.c

```diff
@@ -170,11 +170,18 @@ static void update_recv_state_big_cleared(const struct bt_bap_broadcast_sink *si
 	    reason == BT_HCI_ERR_TERM_DUE_TO_MIC_FAIL) {
 		/* Sync failed due to bad broadcast code */
 		mod_src_param.encrypt_state = BT_BAP_BIG_ENC_STATE_BAD_CODE;
+	} else if (reason == BT_HCI_ERR_CONN_TIMEOUT ||
+		   reason == BT_HCI_ERR_LOCALHOST_TERM_CONN ||
+		   reason == BT_HCI_ERR_REMOTE_USER_TERM_CONN) {
+		/* BIS is not synced anymore */
+		mod_src_param.encrypt_state = BT_BAP_BIG_ENC_STATE_NO_ENC;
 	} else {
 		mod_src_param.encrypt_state = recv_state->encrypt_state;
 	}
 
-	if (reason != BT_HCI_ERR_LOCALHOST_TERM_CONN) {
+	if (reason != BT_HCI_ERR_CONN_TIMEOUT &&
```

Comment:
Can `BT_HCI_ERR_CONN_TIMEOUT` happen when attempting to sync to the BIG? Or only after a successfull sync? 

I'm wondering if the logic should be:
If `mod_src_param.subgroups[i].bis_sync` is set for any subgroup, then we consider it synced and set it to 0. 
If `mod_src_param.subgroups[i].bis_sync` is 0, then we consider it non-synced and set it to `BT_BAP_BIS_SYNC_FAILED`.

Reply:
My concern is that it's not particular explicit in the core spec what termination reason is sent when. The point of my comment was to achieve the same solution but in a way that won't be affected by controllers that may have different behavior

For example, if the BIG Sync fails, then some controller may perhaps send BT_HCI_ERR_CONN_TIMEOUT as the reason, in which case BT_BAP_BIS_SYNC_FAILED should be set